docs/zh/reference/sql/dql/SELECT_STATEMENT.md

```diff
@@ -111,21 +110,21 @@ TableAsName
+| [`JOIN` Clause](../dql/JOIN_CLAUSE.md)         | **``✓``** | **``✓``**    | **``✓``**    | 表示数据来源多个表JOIN。OpenMLDB目前仅支持LAST JOIN。在Online Serving时，需要遵循[Online Serving下LAST JOIN的使用规范](https://openmldb.ai/docs/zh/main/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.html#online-servinglast-join)                                                        |
+| [`WHERE` Clause](../dql/WHERE_CLAUSE.md)       | **``✓``** | **``✓``**    |              | Where 子句用于设置过滤条件，查询结果中只会包含满足条件的数据。                                                                                                                                                                                                                                          |
+| [`GROUP BY` Clause](../dql/GROUP_BY_CLAUSE.md) | **``✓``** |              |              | Group By 子句用于对查询结果集进行分组。分组表达式列表仅支持简单列。                                                                                                                                                                                                                                      |
+| [`HAVING` Clause](../dql/HAVING_CLAUSE.md)     | **``✓``** |              |              | Having 子句与 Where 子句作用类似.Having 子句过滤 GroupBy 后的各种数据，Where 子句在聚合前进行过滤。                                                                                                                                                                                                        |
+| [`WINDOW` Clause](../dql/WINDOW_CLAUSE.md)     | **``✓``** |              | **``✓``**    | 窗口子句用于定义一个或者若干个窗口。窗口可以是有名或者匿名的。用户可以在窗口上调用聚合函数来进行一些分析型计算的操作（```sql agg_func() over window_name```)。在Online Serving时，需要遵循[Online Serving下Window的使用规范](https://openmldb.ai/docs/zh/main/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.html#online-servingwindow) |
+| [`LIMIT` Clause](../dql/LIMIT_CLAUSE.md)       | **``✓``** | **``✓``**    | **``✓``**    | Limit子句用于限制返回的结果条数。目前Limit仅支持接受一个参数，表示返回数据的最大行数。                                                                                                                                                                                                                            |
+| `ORDER BY` Clause                              |           |              |              | 标准SQL还支持Order By子句。OpenMLDB目前尚未支持Order子句。例如，查询语句`SELECT * from t1 ORDER BY col1;`在OpenMLDB中不被支持。                                                                                                                                                                            |
 
 ```{warning}
 在线模式或单机版的select，可能无法获取完整数据。
-因为一次查询可能在多台tablet server上进行大量的扫描，为了tablet server的稳定性，单台tablet server限制了最大扫描数据量，即`scan_max_bytes_size`。
+因为一次查询可能在多台tablet 上进行大量的扫描，为了tablet 的稳定性，单台tablet 限制了最大扫描数据量，即`scan_max_bytes_size`。
```

Comment:
@dl239 the tablet server means the physical server used for tablets? Because we may have multiple tablets running on a physical server, the parameter `scan_max_bytes_size` is for a single tablet, or for the physical server with all tablets?

Reply:
=>
因为一次查询可能在多个 tablet 上进行大量的扫描，为了 tablet 的稳定性，单个 tablet 限制了最大扫描数据量，即`scan_max_bytes_size`。

Reply:
ok~